.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,75 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Bug report
+description: Problems with the software
+title: "[Bug] "
+labels: ["type/bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you very much for your feedback!
+
+        For suggestions or help, please consider:
+        1. [Dev Mail List](mailto:dev@pulsar.apache.org) ([subscribe](mailto:dev-subscribe@pulsar.apache.org)): This will be your FASTEST way to get help!
```

Comment:
I think this is not correct. Pulsar users shouldn't seek general help on the dev mailing list. 
"This will be your FASTEST way to get help!" doesn't sound good in this context and could be removed.

Reply:
Make sense. Let me provide a follow up later today :)

Reply:
FYI https://github.com/apache/pulsar/pull/17793